docs/tools-release-process.md

```diff
@@ -32,6 +32,19 @@ A tools release is performed by a single person each week. We call this person t
 ## Decide on the next version numbers
 According [versioning-and-release-strategy.md](versioning-and-release-strategy.md) patch version bumps (the last of 3 numbers) should only be used for fixes and updates of references to platform versions. For any change in functionality, the second (minor) part of the version should be bumped and new branch created. Instructions for creating a new release branch are further down on this page.
 
+## Tools release planning
+
+### cordova-create principle
+
+The most important principle is that packages such as `cordova-lib` and `cordova-cli` should not use multiple versions of `cordova-common` through the chain of dependencies. This means that if `cordova-common` is updated it should be released before other packages such as `cordova-fetch`, `cordova-create`, `cordova-lib`, `cordova-cli`, etc.
```

Comment:
Why is this a principle?

Reply:
As I think this is confirmed by @raphinesse in <https://lists.apache.org/thread.html/0c55952271a1287bf0303928904bd3164fdb321b7d910346ffaa2d7f@%3Cdev.cordova.apache.org%3E>:
> We do need to ensure that we use the exact same
> cordova-common version everywhere.

I recall from recent discussions that `cordova-common` is used as a singleton since it has a singleton somewhere (I forget where). It would take me some time to dig this information up. If @raphinesse or anyone else can recall this information more easily it would be really appreciated.

Reply:
Yes, cordova-common uses the Singleton pattern at least for events and the logger. To ensure that there's actually only a single Singleton used by all depending packages, they all have to depend on the same version of common.

Ugly, but that's the way it is right now.